rootfs/Dockerfile

```diff
@@ -7,8 +7,11 @@ RUN adduser --system \
 	--group \
 	git
 
-RUN apt-get update \
+RUN apt-key adv --keyserver keyserver.ubuntu.com --recv-keys E1DF1F24 \
+	&& echo "deb http://ppa.launchpad.net/git-core/ppa/ubuntu xenial main" >> /etc/apt/sources.list \
+	&& apt-get update \
 	&& apt-get install -y \
+		git \
```

Comment:
that's true but then again the apt repositories in ubuntu weren't updated in a long time so we had no way of upgrading to 2.8.4 without pinning and installing from source before. Just more concerned about history repeating itself. Anyways still LGTM